plugins/catalog-backend/src/database/DefaultProcessingDatabase.test.ts

```diff
@@ -241,6 +241,7 @@ describe('DefaultProcessingDatabase', () => {
           .select();
         expect(savedRelations.length).toBe(1);
         expect(savedRelations[0]).toEqual({
+          id: savedRelations[0].id,
```

Comment:
This is testing that `savedRelations[0].id` == `savedRelations[0].id`, which is a tautology. Would it be more appropriate to use a different assertion here? Maybe something like (not sure on the type matcher here):
```js
expect(savedRelations[0]).toEqual({
          id: expect.stringMatching(/\d+/),
          "...": "..."
});
```

Reply:
Yup, we can change this to match any number of digits

Reply:
updated